.github/workflows/semgrep.yml

```
@@ -21,4 +22,4 @@ jobs:
       image: semgrep/semgrep
     steps:
       - uses: actions/checkout@v4
-      - run: semgrep ci
+      - run: semgrep ci && true
```

`&&` would cause `true` not to be run
```suggestion
      # discard failures in semgrep
      - run: semgrep ci; true
```